core/arch/arm/plat-rockchip/common.h

```diff
@@ -46,4 +46,7 @@
 #define BITS_WITH_WMASK(bits, msk, shift) \
 				(SHIFT_U32(bits, shift) | BITS_WMSK(msk, shift))
 
+#define __sramfunc __attribute__((section(".sram.text"))) \
```

Comment:
Sorry, I miss the definition here. I will use them for sram data, rodata.
#define __sramdata __attribute__((section(".sram.data")))
#define __sramconst __attribute__((section(".sram.rodata")))